.github/workflows/pre_release_version_bump.yml

```diff
@@ -0,0 +1,40 @@
+name: Manually Triggered Version Bumps
+
+on:
+  push:
+    branches:
+      - auto_vb
```

Comment:
Note, these lines will be removed before merging this into main as these were used for debugging and testing purposes